hoodie-cli/src/main/java/com/uber/hoodie/cli/HoodiePrompt.java

```diff
@@ -31,13 +31,13 @@ public String getPrompt() {
             case INIT:
                 return "hoodie->";
             case DATASET:
-                return "hoodie:" + HoodieCLI.tableMetadata.getTableName() + "->";
+                return "hoodie:" + HoodieCLI.tableMetadata.getTableConfig().getTableName() + "->";
```

Comment:
or wrappers like `HoodieCLI.getTableConfig()` and `HoodieCLI.getActiveTimeline()`?